scripts/languageserver/parse.jl

```diff
@@ -0,0 +1,243 @@
+type VarInfo
+    t
+    doc::String
+end
+
+type Block
```

Comment:
Could you add a comment that describes what a ``Block`` is? It is not clear to me right now :)